Mathlib/Analysis/Meromorphic/Basic.lean

```diff
@@ -52,13 +52,20 @@ theorem MeromorphicAt.eventually_eq_zero_or_eventually_ne_zero {f : 𝕜 → E}
 
 namespace MeromorphicAt
 
+variable
+  {ι : Type*} {s : Finset ι} {F : ι → 𝕜 → 𝕜} {G : ι → 𝕜 → E}
+
 @[fun_prop]
 lemma id (x : 𝕜) : MeromorphicAt id x := analyticAt_id.meromorphicAt
 
-@[fun_prop]
+@[fun_prop, simp]
 lemma const (e : E) (x : 𝕜) : MeromorphicAt (fun _ ↦ e) x :=
   analyticAt_const.meromorphicAt
 
+@[fun_prop, simp]
```

Comment:
That lemma seems unusual; there's no analogue for e.g. analytic, continuous or differentiable functions (as far as I can tell). Let me get a second opinion on zulip about this.
Do you have a particular reason for including this lemma/ do you need it sometime later?